martian-derive/README.md

```diff
@@ -114,8 +114,8 @@ impl ::martian::AsMartianPrimaryType for Chemistry {
 
 ```rust
 impl ::martian::MartianFileType for TxtFile {
-    fn extension() -> &'static str {
-        "txt"
+    fn extension() -> String {
+        "txt".into()
```

Comment:
This is to make the docs consistent with the actual trait.  I wish the docs were correct here, but they aren't.